packages/Takeover/README.md

```diff
@@ -1,31 +1,11 @@
-# Takeover
```

Comment:
why are you removing the basic Readme example? any specific need to do this? the change log should be a different file

Reply:
Ah wrong file removed.